### ISSUE TITLE
Release 2.31, coming soon?

### ISSUE THREAD

**Issue by rhpvorderman** [intel/isa-l#202]
It's been over a year since 2.30 was released (November 2020). 2.31 contains some bug fixes for zlib headers as well as some build fixes in the shell scripts. It also generates a nice `isa-l.h` header file for Windows now.

These are all very welcome updates for [the bindings for python](https://github.com/pycompression/python-isal). I'd love to integrate that in the next release. 

Is there any indication when this will be landing? No pressure, I am just curious when I can plan the next release for python-isal or if I should do an interim release.

**Comment by gbtucker** [comment]
Hi @rhpvorderman, sorry for the delay.  I'll see if I can get one on the schedule and get back to you.

**Comment by rhpvorderman** [comment]
Thanks!

**Comment by rhpvorderman** [comment]
Hi @gbtucker are there any major blockers for the 2.31 release? Again, no rush, I am just trying to get a sense of the schedule so I can decide whether to release python-isal 1.0.0 with the 2.30 release or the 2.31 release embedded.

**Comment by rhpvorderman** [comment]
It turned out to be simple enough to patch the 2.30 release:
- Checkout from the v2.30.0 tag
- Cherry-pick the zlb/gzip headers bugfix commit 
- Checkout the tools directory and Makefile.nmake from the master branch.
- Remove the mem_zero_detect_avx2 and mem_zero_detect_avx512 build from the nmake file.
The result is here: https://github.com/intel/isa-l/compare/v2.30.0...rhpvorderman:isa-l:python-isal2.30withpatches

This is now statically linked in the new python-isal release which I made today: https://github.com/pycompression/python-isal/releases/tag/v1.0.0 .

I started on python-isal two years ago, never having touched a line of C code before in my life. Thanks to creating bindings for ISA-L I have learned C and found that I really enjoy working in that language. Thanks Intel for making this possible by open-sourcing this library. And a special thanks to you @gbtucker for being so helpful along the way, it really is a joy to be working with this library thanks to you.
